### ISSUE TITLE
The MODIS grid seems to be shifted leading to wrong results returned by the getTile()

### ISSUE THREAD

**Issue by zozlak** [MatMatt/MODIS#59]
I was searching for MODIS tiles intersecting with my regions of interest
```r
bbox = matrix(c(39.995535714, 40.995535714, -0.995535714, 0.004464286), nrow = 2, ncol = 2, byrow = TRUE)
getTile(bbox)
```
The result should be h21v08, h21v09, h22v08 & h22v09 but only last two are returned.

While investigating the `getTile()` function I exported the MODIS grid used by the package (the `sr` variable in https://github.com/MatMatt/MODIS/blob/master/R/getTile.R#L277) and compared it in the QGIS with a few MODIS HDFs (MOD09A1.006 from 2018-11-17 tiles h21v08, h21v09, h22v08 & h22v09). The result suggests the grid used by the package is a little shifted comparing to actual tiles location (white box - my region of interest, red lines - package grid, colorful tiles - actual MODIS HDFs coverage):

![grid](https://user-images.githubusercontent.com/6503177/50159774-c6ea9f80-02d7-11e9-8c51-b700dd9b9c23.png)

Is there a reason for that?
How can I properly get the MODIS tiles my region intersects with?

**Comment by fdetsch** [comment]
I agree that this needs further investigation. The boundaries in `MODIS:::sr` and actual G-Polygon coordinates from the granules' metadata differ quite a bit. Until this is fixed, one possible workaround would be to extend your target spatial extent, for example by 0.5 degree. This makes sure that all adjacent tiles are downloaded as well. Finally, just `crop()` the downloaded and extracted images using your original extent as input.

```r
MODISoptions(outProj = "+init=epsg:4326")

## extend target extent by .5 deg
bbox = extent(c(39.995535714, 40.995535714, -0.995535714, 0.004464286))
bbox_xtd = extend(bbox, .5) 

## download and extract images
tfs = runGdal(product = "MOD09A1", collection = NULL
              , begin = as.Date("2018-11-17"), end = as.Date("2018-11-17")
              , extent = bbox_xtd
              , SDSstring = "1100000000000", job = "issue#59"
              , overwrite = TRUE)
tfs = unlist(tfs, use.names = FALSE)

## import images and clip to target extent
rst = stack(tfs)
crp = crop(rst, bbox, snap = "out")
```

![ndvi](https://i.imgur.com/1VBOkla.png)

**Comment by zozlak** [comment]
@fdetsch thanks for the advice.
